symfony/client/src/orangehrmAttendancePlugin/pages/ViewEmployeeAttendanceDetailed.vue

```diff
@@ -330,3 +332,10 @@ export default {
   },
 };
 </script>
+<style scoped lang="scss">
+.orangehrm-paper-container--viewemployee {
+  ::v-deep(.card-header-slot-content .oxd-table-cell-actions) {
```

Comment:
:thinking:

Reply:
need to move action buttons on the right

Reply:
this can be solved with slots I think. Check in the [storybook](https://rajithakumara-oxd.netlify.app/?path=/story/example-cardtable--table-slots&globals=measureEnabled:false)

Reply:
but this is a action table  I think
https://github.com/Ajiharan007/orangehrm/blob/OHRM5X-1184-Add-other-module's-entities/symfony/client/src/orangehrmAttendancePlugin/pages/view-my-attendance.scss
I refer from here